lib_i2c/api/i2c.h

```diff
@@ -386,16 +386,14 @@ extends client interface i2c_master_if : {
  *                 connect to
  *  \param  n      The number of clients connected
  *  \param  p_i2c  The multi-bit port containing both SCL and SDA.
- *                 You will need to set the relevant defines in i2c_conf.h in
- *                 you application to say which bits of the port are used
+ *                 The bit positions of SDA and SCL are configured using the
+ *                 ``sda_bit_position`` and ``scl_bit_position`` arguments.
  *  \param  kbits_per_second The speed of the I2C bus
  *  \param  sda_bit_position The bit of the SDA line on the port
  *  \param  scl_bit_position The bit of the SCL line on the port
- *  \param  other_bits_mask  The mask for the other bits of the port to use
```

Comment:
You could possibly add a comment in #29 about the port being used in set_port_drive_low() mode (which I think is what the original comment tried to say) but this is only a "nice to have"